app/library/postgres.py

```diff
@@ -209,6 +209,16 @@ async def delete_scene_from_postgres(scene_id: str, session):
     if not delete_scene(scene_id, session):
         raise ValueError("Scene ID not valid")
 
+    # Remove the scene from scenario's scene_ids array
+    scenarios = get_scenarios(session)
+    for scenario in scenarios:
+        if int(scene_id) in scenario.scene_ids:
+            scenario_obj: Scenario = await get_scenario_from_postgres(
```

Comment:
I am confused... why are you getting the `scenario_obj` here? You already have the scenario (`for scenario in scenarios:`), why is there a need to make a second request to postgres to get the same scenario?

Reply:
Because I am being silly. Not needed!